features/money_manager.feature

```diff
@@ -29,4 +29,11 @@ Scenario: Signing in should overwrite old answers with new ones
   When  I am on the Money Manager tool
   And   I answer the questions differently
   And   I sign in
-  Then  I have the latest answers
\ No newline at end of file
+  Then  I have the latest answers
+
+Scenario: Passing a special parameter in the url should show all content
```

Comment:
Hi @scott-ad-riley is it worth tagging this scenario to explicitly communicate the fact that it's designed to work with/without JS?

```
@no-javascript
Scenario: Passing a special parameter in the url should show all content
.
.
```

Reply:
I'll add it in.